packages/proxy/src/providers/google.test.ts

```diff
@@ -1480,6 +1481,145 @@ describe("googleCompletionToOpenAICompletion", () => {
       max_depth: 3,
     });
   });
+
+  it("should handle inlineData (image) in response via callProxyV1", async () => {
+    const { json } = await callProxyV1<
+      OpenAIChatCompletionCreateParams,
+      OpenAIChatCompletion
+    >({
+      body: {
+        model: "gemini-3-pro-image-preview",
+        messages: [
+          { role: "user", content: "Generate a small solid red square image" },
+        ],
+        stream: false,
+      },
+    });
+
+    const result = json();
+    expect(result.choices).toHaveLength(1);
+    expect(result.choices[0].message.role).toBe("assistant");
+    expect(result.choices[0].finish_reason).toBe("stop");
+
+    // Response should contain image content
+    const content = result.choices[0].message.content;
+    expect(Array.isArray(content)).toBe(true);
+    if (Array.isArray(content)) {
+      const imageContent = content.find((part) => part.type === "image_url");
+      expect(imageContent).toBeDefined();
+      expect(imageContent?.image_url?.url).toMatch(
+        /^data:image\/(png|jpeg);base64,/,
+      );
+    }
+  });
+
+  it("should handle mixed text and inlineData in response", () => {
```

Comment:
you can either delete this one and the following one or update the examples to do a real call. i'd be curious what a multi-turn would look like